galley/pkg/config/analysis/analyzers/analyzers_test.go

```diff
@@ -506,6 +475,53 @@ func TestAnalyzersHaveDescription(t *testing.T) {
 	}
 }
 
+func setupAndRunCase(tc testCase, cr snapshotter.CollectionReporterFn) (local.AnalysisResult, error) {
+	// Default processing log level is too chatty for these tests
```

Comment:
You could use StopTimer and StartTimer to pause the test timer during the initialization portion of this function (see ://golang.org/pkg/testing/#B.StopTimer).

Reply:
Went ahead and split this function so I could use `b.ResetTimer()`